### DISCUSSION TITLE
Unsupported UI hint: singleline

### DISCUSSION THREAD

**Discussion by michelevioglio97** [elsa-workflows/elsa-core#4933] [Q&A] [answered]
![image](https://github.com/elsa-workflows/elsa-core/assets/133096068/562bfae0-60f0-4ffc-b1dd-64d57152beab)

I followed the steps in the README, but I have lots of errors like the one in the description and in the image. Does anybody knows the simple solution to this kind of problem?

**Comment by glime-ai** [comment, marked as answer]
Based on the context provided, it seems like you are encountering errors when trying to set up or run the Elsa Workflow project. The image link in your question does not lead to an actual image, so I cannot see the specific errors you are referring to. However, here are some general steps you can take to troubleshoot and resolve common issues when setting up a project:

1. **Ensure Dependencies Are Installed**: Make sure that all the necessary dependencies for the project are correctly installed. This includes .NET SDKs, Node.js, npm, and any other libraries or tools required by Elsa Workflow.

2. **Follow the Project's Build Instructions**: Double-check that you are following the build instructions provided in the project's README file or documentation. This may include restoring NuGet packages, building the solution, and running any additional scripts.

3. **Check for Submodule Initialization**: If the project uses Git submodules, ensure that they are properly initialized and updated by running `git submodule update --init --recursive`.

4. **Look for Open Issues**: Check the project's issue tracker on GitHub for any open issues that might be related to the errors you are experiencing. Other users may have encountered the same problems, and there might be solutions or workarounds available.

5. **Seek Help from the Community**: If you cannot resolve the errors on your own, consider reaching out to the community for help. You can ask questions on the project's GitHub Discussions, join the project's Discord server, or post a question on Stack Overflow with the `elsa-workflows` tag.

6. **Check the Environment**: Ensure that your development environment matches the requirements of the project. This includes the operating system, any environment variables, and configuration settings.

7. **Update Your Local Repository**: If you have an older version of the project, try updating your local repository to the latest version. There may have been bug fixes or updates that resolve the issue you are facing.

Since I do not have the specific error message or image, I cannot provide a more targeted solution. If you can provide the exact error messages or describe the issues in more detail, I may be able to offer more specific advice.

**Reply by michelevioglio97** [reply]
The problem was resolved by pulling Elsa Server + Studio and not only Elsa Server as suggested on the first part of the documentation. My bad
